spec/spec_helper.rb

```diff
@@ -93,4 +93,7 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+  config.before(:suite) do
+    Rails.application.load_seed
```

Comment:
I think a better solution is using fixtures
```
Fixtures is a fancy word for sample data. Fixtures allow you to populate your testing database with predefined data before your tests run. Fixtures are database independent and written in YAML. There is one file per model.
```
https://guides.rubyonrails.org/testing.html#what-are-fixtures-questionmark

Reply:
Noice!